recipes/prody/meta.yaml

```diff
@@ -0,0 +1,72 @@
+{% set name = "ProDy" %}
+{% set version = "1.10.10" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/prody/ProDy/archive/v{{ version }}.tar.gz
+  sha256: b1853250d21b05276ca90a938d48daa7fdc3753c2bb82c5c61402b1c705fa880
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  entry_points:
+    - prody = prody.apps:prody_main
+    - evol = prody.apps:evol_main
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+  host:
+    - python
+    - pip
+    - numpy >=1.10
+  run:
+    - libstdcxx-ng  # [linux]
+    - libgcc-ng  # [linux]
```

Comment:
You should not need these directly.  They should be added by the compiler packages